<ui version="4.0" >
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Загрузка РД1</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>импортировать из</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtFileName" />
     </item>
     <item>
      <widget class="QToolButton" name="btnSelectFile" >
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnView" >
       <property name="text" >
        <string>Просмотреть</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>подтверждение</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtPodtv" >
       <property name="text" >
        <string>б/н</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Название поля прикрепления</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtAttach" >
       <property name="text" >
        <string>TMO2</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" >
    <widget class="QCheckBox" name="AccCheck" >
     <property name="text" >
      <string>только текущий счёт</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <widget class="QCheckBox" name="OplataCheck" >
     <property name="text" >
      <string>загружать оплаченные</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0" >
    <widget class="QCheckBox" name="OtkazCheck" >
     <property name="text" >
      <string>загружать отказы</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0" >
    <widget class="QCheckBox" name="chkAttach" >
     <property name="text" >
      <string>загружать данные о прикреплении</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" >
    <widget class="QCheckBox" name="chkOnlyAttach" >
     <property name="text" >
      <string>загружать только данные о прикреплении</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" >
    <widget class="QProgressBar" name="progressBar" >
     <property name="value" >
      <number>24</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="9" column="0" >
    <widget class="QLabel" name="stat" >
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item row="10" column="0" >
    <widget class="QTextBrowser" name="log" />
   </item>
   <item row="11" column="0" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnImport" >
       <property name="text" >
        <string>начать импортирование</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelNum" >
       <property name="text" >
        <string>всего записей в источнике:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose" >
       <property name="text" >
        <string>закрыть</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
